iss/modeller.py

```diff
@@ -110,15 +106,14 @@ def raw_qualities_to_histogram(qualities):
     # quals = util.split_list([i for i in zip(*qualities)], n_parts=cpus)
     cdfs_list = []
     for q in qualities:
-        numpy_log_handler = np.seterrcall(util.nplog)
         with np.errstate(under='ignore', divide='call'):
             try:
                 kde = stats.gaussian_kde(q, bw_method=0.2 / np.std(q, ddof=1))
-            except np.linalg.linalg.LinAlgError as e:
```

Comment:
Why changing the exception type here?

Reply:
I noticed a wrong commit was targeted to this branch, which I just reverted. That one included some more debugging things, which is why I changed the exception type here.